src/server/rdb_load.cc

```diff
@@ -893,14 +954,73 @@ void RdbLoaderBase::OpaqueObjLoader::HandleBlob(string_view blob) {
       res->encoding = OBJ_ENCODING_INTSET;
     }
     pv_->ImportRObj(res);
-  } else if (rdb_type_ == RDB_TYPE_HASH_ZIPLIST) {
-    unsigned char* lp = lpNew(blob.size());
-    if (!ziplistPairsConvertAndValidateIntegrity((const uint8_t*)blob.data(), blob.size(), &lp)) {
-      LOG(ERROR) << "Zset ziplist integrity check failed.";
-      zfree(lp);
+  } else if (rdb_type_ == RDB_TYPE_SET_LISTPACK) {
```

Comment:
see my main comment. I expect to have the rdb file that has SET with listpack encoding. 
also, I think the last argument should be `0` and not `1`